src/styles/blocks/about.scss

```diff
@@ -0,0 +1,45 @@
+.about {
+  margin: 120px 0 110px;
+
+  @include onTablet {
+    margin-bottom: 96px;
+  }
+
+  @include onDesktop {
+    margin-bottom: 120px;
+  }
+
+  &__title {
+    @include title;
+    margin: 0 0 32px;
+    text-align: left;
+
+    @include onTablet {
+      font-size: 40px;
+      line-height: 40px;
+    }
+
+    @include onDesktop {
+      font-size: 44px;
+      line-height: 48px;
+    }
+  }
+
+  &__bullet-points {
+    margin-bottom: 40px;
+
+    &--text {
+      margin: 0;
+      font-weight: 700;
+      line-height: 22px;
```

Comment:
It's good to use variables for line-heights as well to maintain consistency. You can create a variable like $line-height-small: 22px; in your variables.scss file and use it here. 🌟